spec/rubocop/cop/lint/safe_navigation_chain_spec.rb

```diff
@@ -189,6 +189,19 @@
       RUBY
     end
 
+    %w[+ - * /].each do |operator|
```

Comment:
Can you separate each test case instead of the loop? Because the number is limited.